Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -353,6 +353,23 @@ TEST_CASE(test_jbig2_generic_region_arithmetic_code)
             EXPECT_EQ(frame.image->get_pixel(x, y), bmp_frame.image->get_pixel(x, y));
 }
 
+TEST_CASE(test_jbig2_generic_region_arithmetic_code_tpgdon)
```

Comment:
I don't expect any change, this is simply a question:
Why don't you add the test image in the same commit as the test using it? Both of them don't serve any purpose without the other, so it makes more sense to me to add them together.

Reply:
Because the test image has a long description of how I created it. That seems a bit tangential to the actual feature.

(Also, it makes it easy to cherry-pick just the image to different branches locally, without getting merge conflicts.)

Reply:
Ok, thanks for the answer